library/src/scripts/toolTip/ToolTip.tsx

```diff
@@ -90,7 +90,7 @@ function TriangleTooltip(props: { children: React.ReactNode; label: string; aria
  *
  * Custom children (not base dom nodes), must use React.forwardRef().
  */
-export function ToolTip(props: { children: React.ReactNode; label: string; ariaLabel?: string }) {
+export function ToolTip(props: { children: React.ReactNode; label: React.ReactNode; ariaLabel?: React.ReactNode }) {
```

Comment:
```suggestion
export function ToolTip(props: { children: React.ReactNode; label: React.ReactNode; ariaLabel?: string}) {
```

This should be a string.